extensions/mssql/src/mssql.d.ts

```diff
@@ -1300,6 +1300,23 @@ declare module 'mssql' {
 			 */
 			schema: string | undefined;
 		}
+
+		export interface Database extends SqlObject {
```

Comment:
what about the other advanced options?

Reply:
I'll be adding those in the next release. We wanted to get a basic version out first, since it's still very useful even with a small number of options.